bodhi/server/util.py

```diff
@@ -491,6 +491,7 @@ def composestate2html(context, state):
         'punging': 'warning',
         'notifying': 'warning',
         'success': 'success',
+        'cleaning': 'success',
```

Comment:
Let's use the warning class here. The name warning is weird, but we're really using it to get things to be orange/yellow in the UI and the cleaning is running so we want to indicate that things aren't done yet.